bitnami/tomcat/templates/_helpers.tpl

```diff
@@ -61,7 +61,13 @@ Expand the name of the chart.
 Return the proper CATALINA_OPTS value
 */}}
 {{- define "tomcat.catalinaOpts" -}}
-  {{- printf "%s %s" .Values.catalinaOpts .Values.metrics.jmx.catalinaOpts  | trim  -}}
+  {{- if and .Values.catalinaOpts .Values.metrics.jmx.enabled -}}
+    {{- printf "%s %s" .Values.catalinaOpts .Values.metrics.jmx.catalinaOpts  | trim  -}}
+  {{- else if .Values.catalinaOpts -}}
+    {{- printf "%s" .Values.catalinaOpts | trim  -}}
+  {{- else if .Values.metrics.jmx.enabled -}}
+    {{- printf "%s" .Values.metrics.jmx.catalinaOpts  | trim  -}}
+  {{- end -}}
```

Comment:
Why don't we simple use "cat" to concatenate both strings? See http://masterminds.github.io/sprig/strings.html
```suggestion
  {{- cat .Values.catalinaOpts .Values.metrics.jmx.catalinaOpts | trim  -}}
```

Reply:
Because when you use catalinaOpts with metrics.jmx.enabled=false, metrics.jmx.catalinaOpts will be concated anyway and it is prefilled.

Reply:
Done.

Reply:
Seems like i little messed it up with rebase.. Should i close this and open new clean PR? :)